test/app/app.rb

```diff
@@ -24,4 +24,4 @@
 org = Organization.create!(name: 'test')
 org.users.create!(email: 'squirrel@example.com')
 
-User.find_by(email: 'squirrel@example.com')
+User.find_by(email: 'squirrel@example.com') # bump
```

Comment:
Table Scan: The database reads **100%** (5000) of the rows in **users**, skipping any indexes.